packages/create-block-tutorial-template/src/index.js

```diff
@@ -1,8 +1,8 @@
 module.exports = {
 	defaultValues: {
-		slug: 'block-template-gutenpride',
-		namespace: 'gutenpride',
-		title: 'Gutenpride',
+		slug: 'create-block-tutorial-template',
+		namespace: 'create-block-tutorial-template',
+		title: 'Create Block Tutorial Template',
```

Comment:
Should we live those values more customized so it better aligns with the tutorial?
```suggestion
		slug: 'gutenpride',
		namespace: 'create-block-tutorial',
		title: 'Gutenpride',
```

Reply:
yeah, I think that makes sense! I'll change that.

Reply:
fixed and pushed!